styles/roblox/catppuccin.user.css

```diff
@@ -0,0 +1,323 @@
+/* ==UserStyle==
+@name Roblox Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/roblox
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/roblox
+@version 0.1.0
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/roblox/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Aroblox
+@description Soothing pastel theme for Roblox
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire*", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document @-moz-document domain('www.roblox.com') {  
```

Comment:
```suggestion
@-moz-document domain('www.roblox.com') {  
```

Reply:
Sometimes My Genius... It's Almost Frightening